lib/full-site-editing/block-templates.php

```diff
@@ -105,6 +105,39 @@ function _gutenberg_get_template_files( $template_type ) {
 	return $template_files;
 }
 
+/**
+ * Parses wp_template content and injects the current theme's
+ * stylesheet as a theme attribute into each wp_template_part
+ *
+ * @param string $template_content serialized wp_template content.
+ * @param string $theme the active theme's stylesheet.
+ *
+ * @return string Updated wp_template content.
+ */
+function _inject_theme_attribute_in_content( $template_content, $theme ) {
+	$has_updated_content = false;
+	$new_content         = '';
+	$template_blocks     = parse_blocks( $template_content );
+
+	foreach ( $template_blocks as $key => $block ) {
+		if ( 'core/template-part' === $block['blockName'] && ! isset( $block['attrs']['theme'] ) ) {
+			// phpcs:ignore
+			$template_blocks[$key]['attrs']['theme'] = $theme;
+			$has_updated_content                     = true;
```

Comment:
`composer run format` is your friend if you are unsure 😄 Riad is right, you will need spaces around `$key`.

```suggestion
			$template_blocks[ $key ]['attrs']['theme'] = $theme;
			$has_updated_content                       = true;
```

Reply:
> composer run format is your friend if you are unsure 😄 Riad is right, you will need spaces around $key.

Thank you! 🙏